test-suite/src/test/scala/org/scalajs/testsuite/javalib/FormatterTest.scala

```diff
@@ -189,6 +189,18 @@ object FormatterTest extends JasmineTest {
       expectF("%f",      new JDouble(3e10)).toEqual("30000000000.000000")
       expectF("%04f",    new JDouble(Double.NaN)).toEqual(" NaN")
     }
+    
+    it("should support ',' flag") {
+      expectF("%,d",   new Integer(0)).toEqual("0")
+      expectF("%,d",   new Integer(5)).toEqual("5")
+      expectF("%,d",   new Integer(50)).toEqual("50")
+      expectF("%,d",   new Integer(500)).toEqual("500")
+      expectF("%,.2f",   new JFloat(500)).toEqual("500.00")
+      
+      expectF("%,d",   new Integer(5000)).toEqual("5,000")
+      expectF("%,d",   new Integer(500000)).toEqual("500,000")
+      expectF("%,.2f",   new JFloat(5000)).toEqual("5,000.00")
```

Comment:
Please add tests for cases with leading zero padding (to show that it is not grouped, e.g. `f"${1234}%,012.2f" = 00001,234.00`)